### PR TITLE
Bug 1130652 - Add user review guidelines- adding placeholder text

### PR DESCRIPTION
**Pull request by gurumukhi**

### CODE REVIEW

File: src/templates/ratings/add.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
       </p>
       <p class="rating-label">{{ _('Please select a star rating.') }}</p>
       <textarea id="review-body" rows="2" cols="40" name="body"
-                placeholder="{{ _('Tell us what you love about this app!') }}"
+                placeholder="{{ _('Tell us what you like about this app. Write like you’re telling a friend about your experience with the app. Give specific details, for example, What features did you like or dislike?') }}"
```
Comment (chuckharmston) [inline src/templates/ratings/add.html]: We should probably add an L10n comment here to tell the localizers that this is the instruction for a text box.

Comment (chuckharmston) [inline src/templates/ratings/add.html]: Also, the "W" in "What" shouldn't be capitalized.

**Comment by chuckharmston** [comment]
Can you squash the two commits?

**Comment by chuckharmston** [comment]
r+wc

**Comment by ngokevin** [comment]
can you rebase?

**Comment by ngokevin** [comment]
i'm going to close this. i don't think your branch is recoverable without some git-fu. i'd recommend starting over. thanks for trying :)

**Comment by ngokevin** [comment]
make sure to use --rebase when pulling from master

**Comment by gurumukhi** [comment]
Created other PR - https://github.com/mozilla/fireplace/pull/1354

**Pull request closed without merging**